public/video-ui/src/components/FormFields/TagPicker.js

```diff
@@ -298,6 +308,76 @@ export default class TagPicker extends React.Component {
     }
   }
 
+  onSort = (sortedList) => {
+    const newTagValues = [];
+
+    sortedList.forEach(sortedValue => {
```

Comment:
this could be a [`reduce`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce)